### PR TITLE
[WIP] Eval dockerfile

### PR DESCRIPTION
**Pull request by msaroufim**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Starting from a Python 3.8 image
+FROM python:3.8
```
Comment (weiweiy) [inline docker/Dockerfile:2]: Curious why python 3.8, not more recent version?

**Review by weiweiy — APPROVED** [review]
@artidoro were you able to test this?

File: docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+RUN git clone https://github.com/stanford-crfm/helm /helm
+WORKDIR /helm
+RUN pip install .
+
+# Installing torch, torchvision, and torchaudio
+RUN pip3 install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu118
+
+# Set the task and model
+# Add model to Huggingface like so https://github.com/stanford-crfm/helm/blob/main/docs/huggingface_models.md
+RUN echo 'entries: [{description: "mmlu:subject=philosophy,model=huggingface/gpt2", priority: 1}]' > run_specs.conf
```
Comment (msaroufim) [inline docker/Dockerfile:20]: 1 option is to print an entry per submission with codegen

File: docker/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+# Run helm
+# Results will be in benchmark_output/v1
+RUN helm-run --conf-paths run_specs.conf --suite v1 --max-eval-instances 10 && \
```
Comment (msaroufim) [inline docker/Dockerfile:24]: another option for the final eval infra, for each model submission we create a new suite name and then make sure they're all in the same shared volume

File: docker/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    helm-summarize --suite v1
+
+# Start the web server
+CMD ["helm-server"]
```
Comment (msaroufim) [inline docker/Dockerfile:28]: run this outside of this docker container

**Pull request closed without merging**